pkg/services/auth/auth_token.go

```diff
@@ -35,6 +35,19 @@ func (s *UserAuthTokenService) Init() error {
 	return nil
 }
 
+func (s *UserAuthTokenService) ActiveAuthTokenCount() (int64, error) {
```

Comment:
A bit picky maybe but my OCD is too strong. All other methods in this service are referred to by `Token` and not `AuthToken`.